docs/pages/eas/workflows/pre-packaged-jobs.mdx

```diff
@@ -746,6 +746,25 @@ jobs:
 
 </Collapsible>
 
+<Collapsible summary="Using Maestro prefixed environment variables">
```

Comment:
@sjchmiela, I ended up adding an example so that developers can get a quick reference how to use these variables in a workflow file. Just linking to Maestro documentation wasn't sufficient as to where the environment variable should go in a workflow file, if someone is new to Workflows. Let me know what you think.